### DISCUSSION TITLE
Account Blocked

### DISCUSSION THREAD

**Discussion by MarkhorDev** [sigma67/ytmusicapi#663] [General]
Recently my Youtube account got deleted due to "Suspected Spam". The only thing I can think of that would have caused this would be my usage of this API in my own custom playlist manager program. Unless reporting a bunch of ads that need reporting counts. I managed to get my account back by saying "I haven't been warned about this and I won't do it again".

I am curious is anyone else has run into this issue? Should this risk be added somewhere here? 

I don't know if I got banned for using this tool at all, or if it was just the amount I was using it, which is maybe more than most. Can they tell if a request has come from a 3rd party tool or is it identical to a request from their client? I am unsure whether to avoid using this tool at all or just reducing my usage.

**Comment by svioletg** [comment]
I haven't run into this, although for my usage I don't have an account directly attached to ytmusicapi, but this sounds like a similar situation as to what's happening with youtube-dl (and by extension yt-dlp) with the whole "sign-in to confirm you're not a robot" error people are getting. Maybe youtube is starting to restrict stuff like this a lot stronger?

**Comment by MarkhorDev** [comment]
I reached out to sigma67 and they said that this has happened to their accounts too, although I don't know if this is recent or not. But as far as they know, you get whitelisted from automated blocking once your account is reinstated, and they have never had any issues with reinstating their accounts. I personally think it would be worth adding a warning somewhere in the documentation or readme that this can happen.
